src/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "http://localhost:3000",
+    "target": "http://localhost:8080",
```

Comment:
to jest do wycofania?

Reply:
tak, npm run start:proxy  nasłuchuje na 3000, a później jest przekierowanie na 8080